packages/main/src/plugin/updater.ts

```diff
@@ -62,6 +62,23 @@ export class Updater {
     this.#updateCheckResult = undefined;
   }
 
+  public async openReleaseNotes(version: string): Promise<void> {
+    console.log('called opennotes');
+    if (version === 'current') {
+      version = app.getVersion();
+    } else if (version === 'latest') {
+      version = this.#nextVersion?.substring(1) ?? '';
+    }
+    console.log(version);
+    const urlVersionFormat = version.split('.', 2).join('.');
+    let notesURL = `https://podman-desktop.io/blog/podman-desktop-release-${urlVersionFormat}`;
+    const response = await fetch(notesURL);
```

Comment:
I always get an error thrown here, even if I hardcode notesURL with an existing one, for example 
```
    notesURL = `https://podman-desktop.io/blog/podman-desktop-release-1.12`;
```

Reply:
@feloy Is the error thrown when clicking on the learn more in the release notes or the version button in the status bar? Could you show the error?

Reply:
By changing this code in `openReleaseNotes`:
```
-   let notesURL = `https://podman-desktop.io/blog/podman-desktop-release-${urlVersionFormat}`;
+   let notesURL = `https://podman-desktop.io/blog/podman-desktop-release-1.12`;
+   try {
      const response = await fetch(notesURL);
      if (!response.ok) {
        notesURL = `https://github.com/containers/podman-desktop/releases/tag/v${version}`;
      }
      await shell.openExternal(notesURL);  
+   } catch (err: unknown) {
+     console.log('err', err);
+   }
```

I get this error in the console:
```
main ↪️ err TypeError: fetch failed
    at node:internal/deps/undici/undici:13178:13
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Updater.openReleaseNotes (/Users/phmartin/Documents/gits/podman-desktop/packages/main/dist/index.cjs:27519:25)
    at async /Users/phmartin/Documents/gits/podman-desktop/packages/main/dist/index.cjs:27575:9
    at async /Users/phmartin/Documents/gits/podman-desktop/packages/main/dist/index.cjs:127502:9
    at async /Users/phmartin/Documents/gits/podman-desktop/packages/main/dist/index.cjs:126653:26
    at async WebContents.<anonymous> (node:electron/js2c/browser_init:2:86078)Caused by: Error
    at internalConnectMultiple (node:net:1118:18)
    at afterConnectMultiple (node:net:1685:7)
```

Reply:
@feloy I wan't able to reproduce your error, but I did change `fetch` to `https.get` to check if the page exists, so lmk if that works for you now

Reply:
@SoniaSandler yes, it works now, thanks